docker/plugins.txt

```diff
@@ -6,13 +6,15 @@ authentication-tokens:1.4
 aws-credentials:1.30
 aws-java-sdk:1.11.995
 bootstrap4-api:4.6.0-3
-bouncycastle-api:2.21
+bootstrap5-api:5.1.0-1
```

Comment:
Is it OK to have bootstrap 4 and bootstrap 5 installed?

Reply:
I don't know, as this is just the updated list generated after I updated the plugins, as per the [tdr-dev-documentation](https://github.com/nationalarchives/tdr-dev-documentation/blob/master/manual/update-jenkins.md#option-2-manually-update-plugins):

```
def pluginList = new ArrayList(Jenkins.instance.pluginManager.plugins)
pluginList.sort { it.getShortName() }.each{
  plugin ->
    println ("${plugin.getShortName()}:${plugin.getVersion()}")
}
```

Reply:
From https://jenkins.tdr-management.nationalarchives.gov.uk/pluginManager/installed it looks like v4 is used by JUnit and v5 is used by ECharts, which is also used by JUnit. If the JUnit plugin is bringing in multiple versions of Bootstrap then hopefully they know that they're compatible! 🙃